crawl-ref/source/describe.cc

```diff
@@ -4347,6 +4367,7 @@ static string _monster_stat_description(const monster_info& mi)
     result << _monster_attacks_description(mi);
     result << _monster_missiles_description(mi);
     result << _monster_spells_description(mi);
+    result << _monster_habitat_description(mi);
```

Comment:
Flip these; otherwise it looks weird when looking at some monsters' descriptions, e.g. Cerebov.